BaseTools/Conf/build_rule.template

```diff
@@ -293,6 +293,9 @@
     <Command.CLANGPDB>
         "$(DLINK)" /OUT:${dst} $(DLINK_FLAGS) $(DLINK_SPATH) @$(STATIC_LIBRARY_FILES_LIST) $(DLINK2_FLAGS)
 
+    <Command.CLANGDWARF>

```

Comment:
Comment on commit message:
This commit message reads like a cover letter. And is indeed identical to the cover letter. The cover letter does not need to change. The commit message does.

Superfluous information for a commit message:
- Why are we doing this?

Missing information for a commit message:
- What behaviour is changed by applying this patch?

Reply:
Thank you for review @leiflindholm. I will update this.